packages/examples/fortune-v2/reputation-oracle/server/src/modules/webhook/webhook.dto.ts

```diff
@@ -0,0 +1,78 @@
+import { ApiProperty, ApiPropertyOptional } from "@nestjs/swagger";
+import { IsBoolean, IsDate, IsEnum, IsNumber, IsString, IsUrl } from "class-validator";
+import { WebhookStatus } from "../../common/decorators";
+import { ChainId } from "@human-protocol/sdk";
+import { JobRequestType } from "../../common/enums/job";
+import { JobMode } from "../../../../../../../sdk/typescript/human-protocol-basemodels/src";
+
+export class WebhookIncomingCreateDto {
+  @ApiProperty()
+  @IsEnum(ChainId)
+  public chainId: ChainId;
+
+  @ApiProperty()
+  @IsString()
+  public escrowAddress: string;
+
+  public status?: WebhookStatus;
+  public waitUntil?: Date;
```

Comment:
not necessary, the dto doesn't need it, if you need it for something else maybe you can extend

Reply:
We are using this properties for createIncomingWebhook(dto: WebhookIncomingCreateDto): Promise<boolean> {} 
Before, for this reason we had used interfaces. 
> maybe you can extend
What do you mean here?

Reply:
Status and waitUntil ar not needed properties for the object you receive in the controller. So why to include them in the dto?
If you need to a have a class/dto or whatever with the other 2 properties, you can extend from this dto, but don't add properties you dont need, actually in this case you remove status and waitUntil and nothing would happen